elements/pf-popover/pf-popover.ts

```diff
@@ -450,7 +442,7 @@ export class PfPopover extends LitElement {
       flip: !(this.noFlip ?? false),
       fallbackPlacements: this.flipBehavior,
     });
-    this._popover?.show();
+    this.focusTrap ? this._popover?.showModal() : this._popover?.show();
```

Comment:
@bennypowers I actually removed that bit of code in my last commit. My thought was to use `showModal` in the `focusTrap` scenario instead of `show` and then just override the styling so that it would look identical to a non-focus trapped popover.

Reply:
yeah that sounds about right